drivers/interrupt_controller/Kconfig.it8xxx2

```diff
@@ -9,3 +9,19 @@ config ITE_IT8XXX2_INTC
 	 instance of the shared interrupt driver. To conserve RAM set
 	 this value to the lowest practical value.
 	 this software interrupt default set on by device tree.
+
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_ITE_IT8XXX2_WUC  := ite,it8xxx2-wuc
+
+config ITE_IT8XXX2_WUC
+	bool "ITE it8xxx2 Wakeup controller (WUC) interface"
+	depends on SOC_IT8XXX2
+	default $(dt_nodelabel_has_compat,wuc1,$(DT_COMPAT_ITE_IT8XXX2_WUC))
+	help
+	 This option enables the wakeup controller interface for IT8XXX2
+	 family.
+	 This is required for KSCAN, UART, eSPI, GPIO etc., interrupt support.
+
+config WUC_LOG_LEVEL
+	int "WUC_LOG_LEVEL"
+	default 1
```

Comment:
Add something like this to `drivers/interrupt_controller/Kconfig` and use it instead, 


https://github.com/zephyrproject-rtos/zephyr/blob/0c830822aee87b350025de35c3ba631ad834059d/drivers/can/Kconfig#L16-L18

Reply:
Done